content/en/docs/genai/_index.md

```diff
@@ -0,0 +1,7 @@
++++
+title = "GenAI"
+description = "GenAI with Kubeflow"
+weight = 10
```

Comment:
Let's add the GenAI use-cases after the Getting Started section, so we have About -> Getting Started -> GenAI.
WDYT @franciscojavierarceo ?
```suggestion
weight = 25
```